src/main/scala/org/antipathy/scoozie/builder/PrepareBuilder.scala

```diff
@@ -1,12 +1,13 @@
 package org.antipathy.scoozie.builder
 
 import com.typesafe.config.Config
-import org.antipathy.scoozie.action.prepare.Prepare
-import scala.collection.immutable._
-import scala.collection.JavaConverters._
 import org.antipathy.scoozie.action.filesystem.{Delete, MakeDir}
+import org.antipathy.scoozie.action.prepare.Prepare
 import org.antipathy.scoozie.exception.UnknownStepException
 
+import scala.collection.JavaConverters._
+import scala.collection.immutable._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)